tests/test_artifacts.py

```diff
@@ -0,0 +1,123 @@
+import os
+import responses
+
+from pyartifactory import ArtifactoryArtifact
+from pyartifactory.models import (
+    ArtifactPropertiesResponse,
+    ArtifactStatsResponse,
+    AuthModel,
+)
+
+URL = "http://localhost:8080/artifactory"
+AUTH = ("user", "password_or_apiKey")
+ARTIFACT_PATH = "my-repository/file.txt"
+ARTIFACT_NEW_PATH = "my-second-repository/file.txt"
+ARTIFACT_SHORT_PATH = "/file.txt"
+LOCAL_FILE_LOCATION = "tests/test_artifacts.py"
+ARTIFACT_PROPERTIES = ArtifactPropertiesResponse(
+    repo="my-repository", path=ARTIFACT_SHORT_PATH, createdBy="myself", uri="my_uri"
+)
+ARTIFACT_STATS = ArtifactStatsResponse(
+    uri="my_uri",
+    downloadCount=0,
+    lastDownloaded=0,
+    remoteDownloadCount=0,
+    remoteLastDownloaded=0,
+)
+
+
+class TestArtifact:
```

Comment:
We should really move these tests out of a class, to avoid aaaaaall the `@staticmethod`